bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -318,6 +318,13 @@ spec:
           - get
           - list
           - watch
+        - apiGroups:
+          - machineconfiguration.openshift.io
+          resources:
+          - machineconfigs
+          verbs:
+          - list
+          - watch
```

Comment:
Curious if you dug into [why watch is needed](https://github.com/openshift/windows-machine-config-operator/pull/1235#discussion_r964225221)?

Reply:
I was able to remove this. not sure why you were seeing this

Reply:
Thanks for investigating